src/utils/base-events.js

```diff
@@ -4,7 +4,7 @@ import { getVideoPlayerType, getVideoPlayerVersion } from './video-player-option
 import { VIEW_EVENT } from '../events.consts';
 
 export const createViewStartEvent = (sourceUrl, baseData, customerOptions) => {
-  const providedData = parseProvidedData(customerOptions?.providedData);
+  const providedData = parseProvidedData(customerOptions?.customData);
```

Comment:
To align naming in the entire code to make it consistent 
```suggestion
  const customData = parseProvidedData(customerOptions?.customData);
```

Reply:
👍